lib/Horde/Crypt/Smime.php

```diff
@@ -553,6 +553,7 @@ public function certToHTML($cert)
             '<strong>' . Horde_Crypt_Translation::t("Issuer") . ':</strong>';
 
         foreach ($details['issuer'] as $key => $value) {
+            if (is_array($value)) $value = implode(', ', $value);
```

Comment:
Please use the same technique like in commit b50832dcb8d97a2b394484d833ce18868b6bafe5 here.